src/core/libmaven/PeakDetector.h

```diff
@@ -136,12 +136,21 @@ class PeakDetector {
 	 */
 	vector<mzSlice*> processCompounds(vector<Compound*> set, string setName);
 
-	static vector<EIC*> pullEICs(mzSlice* slice, std::vector<mzSample*>&samples,
-			int peakDetect, int smoothingWindow, int smoothingAlgorithm,
-			float amuQ1, float amuQ3, int baselineSmoothingWindow,
```

Comment:
int peakDetect is not being used in pullEICs

Reply:
Yes, this PR was just minding its own business. I can remove it as an argument but we will eventually be coming back to these methods for more refactoring anyway.